examples/rendering.rs

```diff
@@ -0,0 +1,74 @@
+//! Draws a clock in the corner of the screen.
+
+#![cfg_attr(feature = "strict", deny(warnings))]
+
+extern crate nalgebra as na;
+extern crate rlbot;
+
+use rlbot::ffi;
+use std::error::Error;
+use std::f32::consts::PI;
+
+fn main() -> Result<(), Box<Error>> {
+    let rlbot = rlbot::init()?;
+
+    let mut match_settings = ffi::MatchSettings::default();
+    let players = ["Leonardo", "Michelangelo", "Donatello", "Raphael"];
+    match_settings.NumPlayers = players.len() as i32;
+    for (i, a) in players.iter().enumerate() {
+        match_settings.PlayerConfiguration[i].Bot = true;
+        match_settings.PlayerConfiguration[i].RLBotControlled = true;
+        match_settings.PlayerConfiguration[i].set_name(a);
+        match_settings.PlayerConfiguration[i].Team = (i % 2) as u8;
+    }
+    rlbot.start_match(match_settings)?;
+
+    let mut packets = rlbot.packeteer();
+
+    // Wait for the match to start. `packets.next()` sleeps until the next
+    // packet is available, so this loop will not roast your CPU :)
+    while !packets.next()?.GameInfo.RoundActive {}
+
+    loop {
+        let packet = packets.next()?;
+        let mut total_ms = (packet.GameInfo.TimeSeconds * 1000.0) as i32;
+        let ms = total_ms % 1000;
+        total_ms -= ms;
+        let sec = total_ms / 1000 % 60;
```

Comment:
It's probably just me, but I always forget precedence rules for stuff like this and opt to add parens for extra clarity.